test/Numeric/CompatSpec.hs

```diff
@@ -0,0 +1,20 @@
+module Numeric.CompatSpec (main, spec) where
+
+import Test.Hspec
+import Numeric.Compat
+
+main :: IO ()
+main = hspec spec
+
+spec :: Spec
+spec = do
+  describe "showFFloatAlt" $
+    it "shows a RealFloat value, always using decimal notation" $ do
+      showFFloatAlt Nothing  (12 :: Double) "" `shouldBe` "12.0"
+      showFFloatAlt (Just 4) (12 :: Double) "" `shouldBe` "12.0000"
```

Comment:
Following "one assertion per test", this should be two test cases.  In this particular case I think it's even more natural to do so:

```haskell
it "shows a RealFloat value, always using decimal notation" $ do
  showFFloatAlt Nothing  (12 :: Double) "" `shouldBe` "12.0"
it "allows to specify the decimal places" $ do
  showFFloatAlt (Just 4) (12 :: Double) "" `shouldBe` "12.0000"
```